src/ComputerPlayer.java

```diff
@@ -4,6 +4,8 @@ public class ComputerPlayer extends Player{
 	
 	public ComputerPlayer(){
 		super("R2D2");
+		myName="R2D2";
```

Comment:
This fix is better made in the Player class.  That's why the name is passed to the super constructor.